### PR TITLE
WIP: Adjustment

### PR DESCRIPTION
**Pull request by davidwessman**
Addresses #230.

- [ ] Change adjustment page to look for exact matches.
- [ ] Adds keyboard shortcuts to make adjustment faster.
- [ ] Allow partial searches of card-numbers on vote_users pages.

### CODE REVIEW

**Comment by navro** [comment]
Wow, nice to have shortcuts!

**Comment by JohanWinther** [comment]
Is it possible to automatically send users a new vote code upon adjustment?
People usually can't find their old vote code from the previous meeting which makes it easier to just send a new one when they arrive.

And also my vision with the quick adjustment is to just have a computer with the card reader in the room and then people can swipe their card and get a confirmation on the screen with their name. That is nobody needs to touch any button or confirm their adjustment, it just happens. For extra security somebody can check the card validity before they are swiping. Maybe it's better to name this user friendly page as "Automatic adjustments" or something along those lines?

**Comment by JohanWinther** [comment]
I managed to create my desired functionality with a really hacky solution (check out my [shortcuts.js](https://github.com/Fysikteknologsektionen/sektionsmote/blob/5ea31696507985e1932501110f521fcf39bdd7fe/app/assets/javascripts/shortcuts.js) and [card.js.erb](https://github.com/Fysikteknologsektionen/sektionsmote/blob/5ea31696507985e1932501110f521fcf39bdd7fe/app/views/admin/searches/card.js.erb)).

In short, whenever a user is searched at the quick adjustments page the adjust button is automatically clicked and the information is sent to a new window.
